### PR TITLE
[IT-3891] More docs

### PR DESCRIPTION
**Pull request by zaro0508**
Add additional documentation for deploying the OC app and remove some comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,8 +221,22 @@ To load secrets from SSM parameter store with overridden SSM parameter names:
...
+## Bootstrap
 
-# Login with the AWS CLI
+There are a few items that need to be manually bootstrapped before deploying the
+OpenChallenges application.
+
+* Add OC [secrets](#Secrets) to either the cdk.json or the AWS System Manager parameter store
+* Create an [ACM certificate for the application](#Certificates) using the AWS Certificates Manager
+* Add the Certificate ARN to the cdk.json
+* Update references to the OC docker containers in [app.py](app.py)
```
Comment (tschaffter) [inline README.md]:
```suggestion
* Update references to the OC Docker images in [app.py](app.py)
```

File: README.md [hunk 2]
```diff
@@ -217,8 +221,22 @@ To load secrets from SSM parameter store with overridden SSM parameter names:
...
+  (i.e. ghcr.io/sage-bionetworks/openchallenges-xxx:<tag>)
```
Comment (tschaffter) [inline README.md]:
```suggestion
  (i.e. `ghcr.io/sage-bionetworks/openchallenges-xxx:<tag>`)
```

File: README.md (hunk 1 above)
Comment (zaro0508) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (zaro0508) [reply README.md]: fixed

**Review by tschaffter — APPROVED** [review]

**Pull request merged**